compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/allow-global-mutation-unused-usecallback.expect.md

```diff
@@ -36,6 +36,9 @@ function Component() {
   }
   return t0;
 }
+function _temp() {
+  window.foo = true;
+}
```

Comment:
this is weird, i'm guessing an artifact of the DropManualMemo rewriting which wasn't previously happening for this fixture since validatePreserveManualMemo is off by default for fixtures.